opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -179,6 +179,13 @@ def _extract_timecode_info(mob):
         timecode = timecodes[0]
         timecode_start = timecode.getvalue('Start')
         timecode_length = timecode.getvalue('Length')
+
+        if timecode_start is None or timecode_length is None:
+            raise otio.exceptions.NotSupportedError(
+                "Unexpected timecode value(s) in mob name: `{}`."
```

Comment:
I was confused by the phrasing of this. Could you say "in mob named" or "in mob with name" so it is clear the timecode isn't in the name itself. Thanks!

Reply:
Done.